src/java.base/share/classes/java/text/CollationElementIterator.java

```diff
@@ -76,17 +76,15 @@
  * <p>
  * Example of the iterator usage,
  * <blockquote>
- * <pre>
- *
- *  String testString = "This is a test";
- *  Collator col = Collator.getInstance();
- *  if (col instanceof RuleBasedCollator) {
- *      RuleBasedCollator ruleBasedCollator = (RuleBasedCollator)col;
- *      CollationElementIterator collationElementIterator = ruleBasedCollator.getCollationElementIterator(testString);
- *      int primaryOrder = CollationElementIterator.primaryOrder(collationElementIterator.next());
- *          :
```

Comment:
I believe this is a typo, and have removed it

Reply:
My guess is that this was meant as a vertical ellipsis, so instead of removing it, you can replace it with `⋮`.

Reply:
Makes sense, replaced it with `\u22ee` as you suggested and it displays `⋮` properly when the docs are compiled.